core/src/main/scala/kafka/tools/GetOffsetShell.scala

```diff
@@ -132,23 +161,85 @@ object GetOffsetShell {
         }
     }
 
-    partitionOffsets.toArray.sortBy { case (tp, _) => tp.partition }.foreach { case (tp, offset) =>
-      println(s"$topic:${tp.partition}:${Option(offset).getOrElse("")}")
+    partitionOffsets.toSeq.sortWith((tp1, tp2) => compareTopicPartitions(tp1._1, tp2._1)).foreach {
```

Comment:
nit: `sortWith((tp1, tp2) => compareTopicPartitions(tp1._1, tp2._1))` -> `sortWith(compareTopicPartitions)`

Reply:
The current compareTopicPartitions method expects TopicPartitions, but we have tuples in the collection.
Should I change compareTopicPartitions to expect tuples?